### ISSUE TITLE
Central Cyanosis

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7121]
**Preferred term label:**
central cyanosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Generalized bluish discoloration of the body and the visible mucous membranes, which occurs due to inadequate oxygenation secondary to conditions that lead to an increase in deoxygenated hemoglobin or presence of abnormal hemoglobin. [PMID: 32644593]

**Parent term (use hpo.jax.org/app)**
HP:0000961 - Cyanosis

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Database cross reference**
[SNOWMEDCT_US:95837007](https://phinvads.cdc.gov/vads/ViewCodeSystemConcept.action?oid=2.16.840.1.113883.6.96&code=95837007)

**Comment by pnrobinson** [comment]
added
